src/main/java/com/infogain/debonair/controllers/RestaurantController.java

```diff
@@ -0,0 +1,100 @@
+package com.infogain.debonair.controllers;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.infogain.debonair.dto.Items;
+import com.infogain.debonair.service.CustomerService;
+import com.infogain.debonair.service.RestaurantService;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+
+/**
+ * Restaurant class has all the endpoints which handles Restaurant operations
+ * 
+ * @author Chetna
+ * @since 21-Apr-2020
+ */
+@RestController
+@RequestMapping("/debonair")
+@Api(value = "/debonair")
+@CrossOrigin(origins = "*")
+public class RestaurantController {
+
+	@Autowired
+	RestaurantService restaurantServices;
+	@Autowired
+	CustomerService customerServices;
+
+	/**
+	 * This method is endpoint to fetch Menu details
+	 * 
+	 * @return ResponseEntity object containing Restaurant Menu & HTTP Status OK
+	 * @since 04-May-2020
+	 */
+	@ApiOperation(protocols = "http", notes = "This is used to get menu items.", value = "Show Menu")
+	@GetMapping(path = "/menu")
+	public ResponseEntity<List<Items>> showMenu() {
+		System.out.println("Show Menu");
+		return new ResponseEntity<List<Items>>(restaurantServices.getMenu(), HttpStatus.OK);
+	}
+
+	/**
+	 * This method is endpoint to process the order placed by used.
+	 * 
+	 * @param items object sent from frontend
+	 * @return ResponseEntity object with HTTP Status Created
+	 * @since 04-May-2020
+	 */
+	@ApiOperation(protocols = "http", notes = "This is used to place customer orders.", value = "Take Order")
+	@PostMapping(path = "/order", consumes = MediaType.APPLICATION_JSON_VALUE)
+	public ResponseEntity<Object> takeOrder(@RequestBody Items items[]) {
+		restaurantServices.processOrder(items);
+		return new ResponseEntity<>(HttpStatus.CREATED);
```

Comment:
If Nothing is returned in ResponseEntity<>(HttpStatus.CREATED); why is the return type ResponseEntity<Object>

Reply:
Resolved